docs/contributing/installing.md

```diff
@@ -0,0 +1,52 @@
+---
+id: install
+title: Contributing
+sidebar_label: Install
+---
+
+# Install
+
+## Prerequisits
+
+### Setup a development environment
+
+1. Install the latest stable versions of [Git](https://git-scm.com/download).
+1. Configure git user credentials for [every respository](https://help.github.com/articles/setting-your-username-in-git/#setting-your-git-username-for-every-repository-on-your-computer) or [single repository](https://help.github.com/articles/setting-your-username-in-git/#setting-your-git-username-for-a-single-repository)
```

Comment:
Same note on this being "recommended" if we even include this. I'm not sure that this is the same method if I'm using the github app, etc

Reply:
removing